Configurations.md

```diff
@@ -10,7 +10,7 @@ reorder_imports = false
 ```
 
 Each configuration option is either stable or unstable.
-Stable options can be used directly, while unstable options are opt-in.
+Stable options can always be used, while unstable ones are only available on a nightly toolchain, and opt-in.
```

Comment:
Thanks for the PR! I think the proposed rewording makes it clearer that unstable options can only be used on nightly. I'd suggest a slight tweak.

```suggestion
Stable options can always be used, while unstable options are only available on a nightly toolchain and must be opted into.
```